sale_input_barcode/models/product_line_mixin.py

```diff
@@ -0,0 +1,73 @@
+# copyright 2022 David BEAL @ Akretion
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class ProductLineMixin(models.AbstractModel):
+    _name = "product.line.mixin"
```

Comment:
@bealdav I would have named it with 'barcode' as it is not a 'product-generic' mixin but 'barcode' oriented.

Moreover, I would have put it in another module as it is not specific to 'sale_input'

Reply:
Ok to change mixin name.

Reply:
For another module why not but if somebody may tells me : which module name, which repo, which use cases'll be implemented.

If no use case, I can add in roadmap section

Reply:
Yes. I told that because of your model description :

"Utilities for any model to deal with product_id field"

So, if you want to use it elsewhere you need to pull this one.

Reply:
I don't see any evident case but If somebody find it one, it'll be easy to make an extraction in another module.
Thanks to point me this option